tests/integration/logical/test_list.py

```diff
@@ -123,3 +123,53 @@ def testList(self):
         """
         command_line = self._MENU + [self._POOLNAME]
         RUNNER(command_line)
+
+
+class List4TestCase(unittest.TestCase):
+    """
+    Test listing volumes in an existing pool with some volumes.
+    """
+    _MENU = ['--propagate', 'filesystem', 'list']
+    _POOLNAME = 'deadpool'
+    _VOLUMES = ['livery', 'liberty', 'library']
+
+    def setUp(self):
+        """
+        Start the stratisd daemon with the simulator.
+        """
+        self._service = Service()
+        self._service.setUp()
+        command_line = ['pool', 'create', self._POOLNAME] \
+            + _DEVICE_STRATEGY.example()
+        RUNNER(command_line)
+
+        command_line = ['filesystem', 'create', self._POOLNAME
+                        ] + self._VOLUMES[0]
+        RUNNER(command_line)
+        command_line = ['filesystem', 'create', self._POOLNAME
+                        ] + self._VOLUMES[1]
+        RUNNER(command_line)
+        command_line = ['filesystem', 'create', self._POOLNAME
+                        ] + self._VOLUMES[2]
+        RUNNER(command_line)
+
+    def tearDown(self):
+        """
+        Stop the stratisd simulator and daemon.
+        """
+        self._service.tearDown()
+
+    def testList(self):
```

Comment:
If you are only going to use ```_MENU``` in one place you might as well get rid of it entirely and just write it out explicitly in first line.